WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/PrefMasterSwitchToolbarView.kt

```diff
@@ -103,7 +104,7 @@ class PrefMasterSwitchToolbarView @JvmOverloads constructor(
                 val titleOff = typedArray.getString(R.styleable.PrefMasterSwitchToolbarView_masterTitleOff)
                 val hintOn = typedArray.getString(R.styleable.PrefMasterSwitchToolbarView_masterHintOn)
                 val hintOff = typedArray.getString(R.styleable.PrefMasterSwitchToolbarView_masterHintOff)
-                val titleContentDescription = typedArray
+                titleContentDescription = typedArray
```

Comment:
I suggest we move this right after the `try {` line and in its own paragraph. Grouping this with the `hintOn` and `hintOff` variable somehow implies that it's going to be used later with them. 

```kt
            try {
                titleContentDescription = typedArray
                        .getString(R.styleable.PrefMasterSwitchToolbarView_masterContentDescription)

                val titleOn = typedArray.getString(R.styleable.PrefMasterSwitchToolbarView_masterTitleOn)
                val titleOff = typedArray.getString(R.styleable.PrefMasterSwitchToolbarView_masterTitleOff)
                val hintOn = typedArray.getString(R.styleable.PrefMasterSwitchToolbarView_masterHintOn)
                val hintOff = typedArray.getString(R.styleable.PrefMasterSwitchToolbarView_masterHintOff)
```

Reply:
You are right! 🙇 Resolved 7f26741